homeassistant/components/splunk/__init__.py

```diff
@@ -1,14 +1,13 @@
 """Support to send data to a Splunk instance."""
 import asyncio
-from http import HTTPStatus
 import json
 import logging
-import time
 
 from aiohttp import ClientConnectionError, ClientResponseError
 from hass_splunk import SplunkPayloadError, hass_splunk
```

Comment:
Why is the library called `hass-splunk`? It doesn't seem to be specific for Home Assistant. I suggest renaming the library. It's not good practice to couple the 3rd party libraries tightly to Home Assistant.

Reply:
Because nothing else avaliable worked with aiohttp and had retry queues. The library was written specially to work as best as it could with Home Assistant. There are other libraries out there that are better suited to other purposes.

This also has nothing to do with this PR.

Reply:
Yes it's a side note. Aiohttp and retry queues isn't specific to Home Assistant. 3rd party libraries should not know or care about Home Assistant specifically. They should just follow good standards in general and Home Assistant will be happy too.